Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs

```diff
@@ -7,6 +7,9 @@
 
```

Comment:
https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L16
Documentation link should point to task portal.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L26
You are not using cancellation token, so either use it somewhere or remove it.